src/main/java/com/benetech/mathshare/Application.java

```diff
@@ -11,7 +13,10 @@
 @EnableAutoConfiguration(exclude={DataSourceAutoConfiguration.class,HibernateJpaAutoConfiguration.class})
 public class Application {
 
-	public static void main(String[] args) {
-		SpringApplication.run(Application.class, args);
-	}
+    private static final Logger LOGGER = LoggerFactory.getLogger(Application.class);
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+        LOGGER.info("Application is running");
```

Comment:
Change 'Application' to 'MathShare', so that we know it's our log

Reply:
Ok